nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -895,6 +980,9 @@ def setup_test_data(self, cfg):
             )
             self._test_dl = self.build_pretraining_data_loader(self._test_ds, consumed_samples)
 
+    def mgpt_wrapper(self):
```

Comment:
Good. Now if you add '@property' to that method and add a method : 
def list_export_subnets(self): 
    return ['mgpt_wrapper'] 
then generic export() would automatically work on this model.

Reply:
@borisfom I tried exporting an ONNX file with the latest changes, and I found that the output ONNX name will be `mgpt_wrapper-<ONNX_FILE_NAME>` rather than `<ONNX_FILE_NAME>` due to this override of list_export_subnets.

Which is kind of unexpected as a user, is there a proper way to get this prefix information or export ONNX with this prefix from the ModelPT interface? Thanks.